web/client/utils/StyleUtils.js

```diff
@@ -12,32 +12,142 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree.
  */
-import assign from 'object-assign';
+import tinycolor from 'tinycolor2';
+import uuidv1 from 'uuid/v1';
+import defaultIcon from '../components/map/openlayers/img/marker-icon.png';
+import isEmpty from 'lodash/isEmpty';
+import { flattenFeatures } from './VectorStyleUtils';
 
-const getColor = function(color) {
-    return `rgba(${color.r}, ${color.g}, ${color.b}, ${color.a})`;
+const getFeatureCollectionSingleGeometryType = ({ features } = {}) => {
```

Comment:
please provide jsdoc, possibly, for all new functions that are present in the various utils files